pom.xml

```diff
@@ -190,8 +184,33 @@
                         <goal>report</goal>
                     </goals>
                 </execution>
-            </executions>
-        </plugin>
+                <execution>
+                  <id>check-coverage</id>
+                  <goals>
+                    <goal>check</goal>
+                  </goals>
+                  <configuration>
+                    <excludes>
+                      <!-- we exclude the GUI classes, because we don't have Swing GUI tests in CI --> 
+                      <exclude>**/support/gui/*</exclude>
+                    </excludes>
+                    <rules>
+                    <rule implementation="org.jacoco.maven.RuleConfiguration">
+                      <element>BUNDLE</element>
+                      <limits>
+                        <limit implementation="org.jacoco.report.check.Limit">
+                          <counter>CLASS</counter>
+                          <value>MISSEDCOUNT</value>
+                          <!-- in theory we would like to have MISSEDCOUNT=0 but in some cases, Jacoco counts one phantom missed instruction, so ze go for one -->
+                          <maximum>7</maximum>
```

Comment:
shouldn't there be a comment for which instructions are phantom missed? This could help in the future, if the build fails.